test/TestProviderRound.spec.js

```diff
@@ -21,13 +21,13 @@ contract('ProviderRound', accounts => {
     it("should register a provider's parameters", async () => {
       await providerRound.provider(22, 10, 1, 25, {from: accounts[0]});
       await providerRound.provider(10, 20, 2, 35, {from: accounts[1]});
-      let firstProvider = await providerRound.providers.call(accounts[0]);
+      const firstProvider = await providerRound.providers.call(accounts[0]);
       assert.equal(firstProvider[0], PROVIDER_REGISTERED);
```

Comment:
Can we not call this like `firstProvider.status` instead of `firstProvider[0]`?

Reply:
Same as above, the field names seem to be lost with web3

Reply:
Alright, if that is the case, then we can merge this.